### ISSUE TITLE
No response when building Multidimension coaster

### ISSUE THREAD

**Issue by telk5093** [OpenRCT2/OpenRCT2#6686]
**OS:** Windows 10
**Version:** 0.1.2
**Commit/Build:** e23c108

There is no response when I build a multidimension coaster (such as "X")
No dump files are generated.
0.1.1 has no such issue.

**Steps to reproduce:**
1. Open any map where multidimension coaster is available in.
2. Build "X".
3. Test it
4. Then game has no response.

**Dump file**
No dump files are created.

**Screenshots / Video:**
![test](https://user-images.githubusercontent.com/10726524/32957962-71e6d7f6-cc00-11e7-9da0-b626913cf5b1.gif)
(Resized for saving filesize)

**Comment by Nubbie** [comment]
Commit: e23c108
Dump too large for github, extracted from task manager

Call stack;
```
>	openrct2.dll!invalidate_sprite_max_zoom(rct_sprite * sprite, int maxZoom) Line 76	C	Symbols loaded.
 	[Inline Frame] openrct2.dll!invalidate_sprite_2(rct_sprite *) Line 116	C++	Symbols loaded.
 	openrct2.dll!vehicle_update_track_motion(rct_vehicle * vehicle, int * outStation) Line 9489	C++	Symbols loaded.
 	openrct2.dll!loc_6DDF9C(Ride * ride, rct_tile_element * tileElement) Line 4983	C	Symbols loaded.
 	openrct2.dll!ride_create_vehicles(Ride * ride, int rideIndex, rct_xy_element * element, int isApplying) Line 4932	C	Symbols loaded.
 	openrct2.dll!ride_is_valid_for_test(int rideIndex, int goingToBeOpen, int isApplying) Line 5381	C	Symbols loaded.
 	openrct2.dll!RideSetStatusAction::Execute() Line 161	C++	Symbols loaded.
 	openrct2.dll!GameActions::Execute(const GameAction * action) Line 178	C++	Symbols loaded.
 	openrct2.dll!ride_set_status(int rideIndex, int status) Line 156	C++	Symbols loaded.
 	[Inline Frame] openrct2.dll!window_event_dropdown_call(rct_window *) Line 1932	C	Symbols loaded.
 	openrct2.dll!input_state_widget_pressed(int x, int y, int state, short widgetIndex, rct_window * w, rct_widget * widget) Line 1247	C	Symbols loaded.
 	openrct2.dll!game_handle_input_mouse(int x, int y, int state) Line 322	C	Symbols loaded.
 	openrct2.dll!game_handle_input(...) Line 129	C	Symbols loaded.
 	openrct2.dll!game_update(...) Line 399	C	Symbols loaded.
 	openrct2.dll!OpenRCT2::Context::Update() Line 780	C++	Symbols loaded.
 	openrct2.dll!OpenRCT2::Context::RunVariableFrame() Line 735	C++	Symbols loaded.
 	[Inline Frame] openrct2.dll!OpenRCT2::Context::RunFrame() Line 670	C++	Symbols loaded.
 	openrct2.dll!OpenRCT2::Context::RunGameLoop() Line 645	C++	Symbols loaded.
 	openrct2.dll!OpenRCT2::Context::RunOpenRCT2(int argc, const char * * argv) Line 167	C++	Symbols loaded.
 	openrct2.dll!NormalisedMain(int argc, const char * * argv) Line 62	C++	Symbols loaded.
 	openrct2.dll!LaunchOpenRCT2(int argc, wchar_t * * argvW) Line 53	C++	Symbols loaded.
 	[External Code]		Annotated Frame
```

Locals;
```
+		sprite	openrct2.dll!0x00007ff977c7b920 {pad_00=0x00007ff977c7b920 "" unknown={sprite_identifier=0 '\0' misc_identifier=...} ...}	rct_sprite *
+		sprite->unknown	{sprite_identifier=0 '\0' misc_identifier=1 '\x1' next_in_quadrant=17 ...}	rct_unk_sprite
		sprite->unknown.sprite_left	807	short
```

**Comment by IntelOrca** [comment]
This can be reproduced by loading Six Flags Magic Mountain and clicking test on the ride: X.

**Comment by SidaRadis** [comment]
This issue also occurs with the Lay-down Roller Coasters designs that are included with the game (Evil Vultures & Mystic Flight).

**Comment by Gymnasiast** [comment]
Fixed in 8027af56b6074d8456f5e0af538c41af11efe7a5.
